pkg/storagepool/storagepool.go

```diff
@@ -162,3 +165,97 @@ func (v *casPoolOperation) Delete() (*v1alpha1.CasPool, error) {
 	}
 	return pool, nil
 }
+
+type storagePoolOperation struct {
+	poolName  string
+	k8sClient *m_k8s_client.K8sClient
+}
+
+func NewStoragePoolOperation(poolName string) (*storagePoolOperation, error) {
```

Comment:
This is an exported function and should have a comment.

Reply:
added